src/neuroglancer/sliceview/volume/segmentation_renderlayer.ts

```diff
@@ -149,6 +166,7 @@ uint64_t getMappedObjectId(uint64_t value) {
 `);
     }
     builder.addUniform('highp uvec2', 'uSelectedSegment');
+    builder.addUniform('highp uvec2', 'uBaseSelectedSegment');
```

Comment:
I don't think there is a need for a separate parameter --- instead `uSelectedSegment` can just be set to the base selected segment if `baseSegmentHighlighting` is enabled.